server/index.js

```diff
@@ -7,7 +7,6 @@ const router = require('./router.js');
 app.use(morgan('dev'));
 app.use(bodyParser.json());
 app.use('/api/about', router);
```

Comment:
Should we make sure everybody is on the same page of /api/<[module]>  in the next standup meeting?

Reply:
Yeah for sure